demo/alloy-showcase-webapp/src/test/java/com/liferay/faces/test/alloy/showcase/datatable/DataTableTesterBase.java

```diff
@@ -238,37 +184,31 @@ protected String getRowSelectionInputXpath(Customer customer) {
 			"')]//input[contains(@id,'customers:')]";
 	}
 
-	/**
-	 * Determines whether or not the specified page number is active in the rendered <code>alloy:paginator</code>
-	 * markup.
-	 */
-	protected ExpectedCondition<Boolean> isPaginatorPageNumberButtonActive(int pageButtonNumber) {
```

Comment:
@ngriffin7a,  I was making it consistent with the `PrimeFacesUsersPortletTester`: https://github.com/liferay/liferay-faces-portal/blob/master/test/integration/src/test/java/com/liferay/faces/portal/test/integration/demo/PrimeFacesUsersPortletTester.java#L580-L582

I'm fine with changing the local variable name to whatever you'd like as a compromise (and I think that's the only time we'd see it in an `if` statement), but I think the method name should remain consistent with the PF tester and with Selenium:
https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/support/ui/ExpectedConditions.html